interactive-etl/setup.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -u 
+set -e
+set -o errexit
+
+NAMESPACE=${1:-flink}
+
+printf "\n\nInstalling example components into namespace: %s\n\n" "$NAMESPACE"
+
+# Install CertManager - this is needed by the Flink Kubernetes Operator
+printf "\nChecking for CertManager install\n"
+if kubectl get namespace cert-manager ; then
```

Comment:
One of the other passing thoughts is should we hard code kubectl or should we allow people to swap between `kubectl` and `oc` ?

Reply:
Yeah I will make it optional with `kubectl` as the default